### DISCUSSION TITLE
Unable to provide values in initialData for TinyMCE dialog

### DISCUSSION THREAD

**Discussion by sha2nktiwade** [tinymce/tinymce#8934] [General]
Hi Team,

We have a use-case where we have developed a custom plugin, which opens up a TinyMCE dialog and has some options along with checkboxes in front of it. It looks like below:
![image](https://github.com/tinymce/tinymce/assets/130130998/5d38f407-6507-4f3f-a17a-c3c4044b8fb8)

- In this I am checking some check-boxes and clicking on OK button. 
- This will put in a div section in the editor. 
- Now I will click on edit button that we are populating through the context menu.
- This edit button will again the TinyMCE dialog similar to the one posted above. 
- The extra thing I want is the to retain the options that were already selected 
- I am trying to do this with initialData of TinyMCE, but unable to access the check-boxes and click them.

Could you please let me know how I would be able to access the check-boxes and click them.

**Comment by TheSpyder** [comment]
Our examples include dialogs with a checkbox and initial data
https://www.tiny.cloud/docs/tinymce/6/dialog-examples/
